src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/ExpressionCompilerTests.cs

```diff
@@ -6435,7 +6435,10 @@ .maxstack  1
             });
         }
 
-        [Fact]
+        // PROTOTYPE(NullableReferenceTypes): Use C#8 to compile expressions.
+        // (Expression below currently reports "CS0453: The type 'object' must be a non-nullable
```

Comment:
>(Expression below currently reports "CS0453: The type 'object' must be a non-nullable [](start = 11, length = 85)

What is causing this?

Reply:
Looks like a bug: `IsFeatureEnable()` fails since there are no syntax trees to check `ParseOptions`.

---
In reply to: [171141496](https://github.com/dotnet/roslyn/pull/25087#discussion_r171141496) [](ancestors = 171141496)

Reply:
> Looks like a bug: IsFeatureEnable() fails since there are no syntax trees to check ParseOptions. Updated the comment.

But it looks like there is something more going on. I don't see the code using "?", nullable types or object explicitly. I would like to understand where the error is coming from

---
In reply to: [171153451](https://github.com/dotnet/roslyn/pull/25087#discussion_r171153451) [](ancestors = 171153451,171141496)

Reply:
The expression being executed contains a lambda with `object? o` parameter (see call to `CompileExpression`).

---
In reply to: [171351376](https://github.com/dotnet/roslyn/pull/25087#discussion_r171351376) [](ancestors = 171351376,171153451,171141496)